src/css/globals.scss

```diff
@@ -1,3 +1,6 @@
 @import 'globals/body';
 @import 'globals/headings';
 @import url('https://fonts.googleapis.com/css?family=Raleway:300,400,400i,500,600,700|Roboto:300,300i,400,400i,500,700');
+
+// dependencies
+@import 'node_modules/@glidejs/glide/src/assets/sass/glide.core';
```

Comment:
should be imported in the component

Reply:
But other components may use glide, SfCarousel for example, then the scss would be imported many times ans css duplicated, right?
I think only functions, variables and mixins may be imported by components with SASS, take a look here:
https://github.com/angular/angular-cli/issues/8431

Reply:
Probably no, webpack will bundle treat all imports from one source as one afaik

Reply:
I think webpack does not handle this by default, maybe there is a postcss plugin to prevent duplicated CSS code, or a cssnano config...
But imo it's not a good practice at all, I think it's global styles, it will be used in several components, it's not specific to gallery.
Why do you think is better to import it inside component SCSS (and repeat the import many times)?

Reply:
Are you sure? It's treated as any other module I guess so should be optimized. Otherwise every css like transitions would be included multiple times

I'm agains adding it to global css since it adds code that is not always used

Reply:
Ok you were right, but i found workaround https://paul.wellnerbou.de/2015/05/18/avoid-multiple-imports-of-the-same-scss-file-with-sass/

Reply:
as we talked, I'm not really convinced of this yet, but I've changed haha